src/App.tsx

```diff
@@ -15,48 +16,125 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum Sortfield {
+  SORT_FILED_ALPHABET = 'Sort alphabetically',
+  SORT_FILED_LENGTH = 'Sort by length',
+}
+
+interface FilterParams {
+  sortField: Sortfield | string;
+  isReversed: boolean;
+}
+
+type IsLightFunction = (a: Sortfield | string, b: Sortfield | string) => string;
+
+function getPrepareGoods(
+  goods: string[],
+  { sortField, isReversed }: FilterParams,
+) {
+  let preparedGoods = [...goods];
+
+  if (sortField) {
+    preparedGoods.sort((good1, good2) => {
+      switch (sortField) {
+        case Sortfield.SORT_FILED_ALPHABET:
+          return good1.localeCompare(good2);
+        case Sortfield.SORT_FILED_LENGTH:
+          return good1.length - good2.length;
+        default:
+          return 0;
+      }
+    });
+  }
+
+  if (isReversed) {
+    preparedGoods = preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+}
+
+const isLightStyle: IsLightFunction = (sortField, sortFieldType) => {
```

Comment:
with classnames library you don't need this function. there is a way to add conditional classes provided by this library itself. you jus need to add object where is properly is class and value is condition when this class should be applied.